lib/WebDriverTargetLocator.php

```diff
@@ -37,6 +37,13 @@ public function defaultContent();
      */
     public function frame($frame);
 
+    /**
+     * Switch to the parent iframe.
+     *
+     * @return WebDriver The driver focused on the given frame.
```

Comment:
Add something like `@todo Add in next major release` - so that we find if when preparing 2.0.